### DISCUSSION TITLE
Problems with the write_html function when changing the color of heading elements

### DISCUSSION THREAD

**Discussion by liviaalvzs** [py-pdf/fpdf2#880] [Q&A] [answered]
Hello! I'm having problems with the write_html function. Whenever I try to plot a heading element, it comes out red, and I can't change its color. Is it a problem with my code or with the lib?
I already tried to set the color in every possible way that I know of:
`pdf.set_text_color(255,255,255);
pdf.write_html(html_text)`
or
`html = '<h1 color="#00ff00>hello in green</h1></font>';
pdf.write_html(html)`
It always plots in red. Someone can help me?

**Comment by Lucas-C** [comment, marked as answer]
Hi @liviaalvzs!

Thank you for getting in touch.

You are right, this currently a limitation of `fpdf2` HTML renderer.
The headings color is hardcoded & set there:
https://github.com/py-pdf/fpdf2/blob/2.7.5/fpdf/html.py#L386

I just merged a PR that adds support for this: https://github.com/py-pdf/fpdf2/pull/886

While waiting for a new `fpdf2` release including this feature,
you can already test it by installing `fpdf2` from `git`:

    pip install git+https://github.com/py-pdf/fpdf2.git@master

**Reply by liviaalvzs** [reply]
Thanks for your help and for being so fast! It worked here.
